src/components/Article.jsx

```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (article) => {
+  const { title, date, text } = article;
+  const today = date;
+  const dateISO = today.toISOString().slice(0, 10);
```

Comment:
```suggestion
export const Article = ({ article }) => {
  const { title, date, text } = article;
  const today = date;
  const dateISO = today.toISOString().slice(0, 10);
```
There is an error because article is actually an object with props.
Try to console.log the values to debug your code
![image](https://github.com/mate-academy/react_article/assets/96409563/2eb14a17-d59b-46af-b6e6-50f865baf548)